seadssite/views.py

```diff
@@ -41,7 +41,7 @@ def register(request):
 
     # if a user "POST"s, check phone number, provider and user/profile form(models)
     if request.method == 'POST':
-        print ("user is about to register")
+        print("user is about to register")
```

Comment:
Yes definitely. What I changed is merely a stylistic space character. It's
beyond my scope of knowledge right now to add a proper logger. Also because
the code works as is, I suggest saving this change for another sprint. It's
not super urgent compared to tasks like setting up the visualization.
Thoughts?


On Tue, Nov 10, 2015 at 5:01 PM, Ian Gudger <notifications@github.com>
wrote:

> In seadssite/views.py
> <https://github.com/seadsystem/website/pull/2#discussion_r44489867>:
>
> > @@ -41,7 +41,7 @@ def register(request):
> >
> >      # if a user "POST"s, check phone number, provider and user/profile form(models)
> >      if request.method == 'POST':
> > -        print ("user is about to register")
> > +        print("user is about to register")
>
> Logging can be really helpful. Can we replace these print statements with
> proper logging that includes a timestamp and preferably the logging
> location?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/seadsystem/website/pull/2/files#r44489867>.
>